### PR TITLE
Add shortcut and fallback when migrating to volto

### PR DESCRIPTION
**Pull request by pbauer**
this is to better handle running the migration multiple times

### CODE REVIEW

**Review by ericof — APPROVED** [review]
LGTM

**Comment by davisagli** [comment]
Hmm, why is the changelog verifier passing when there is no changelog?

File: news/97.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Handle issues that may appear when running multiple times or when plone.volto was installed first. [pbauer]
```
Comment (davisagli) [inline news/97.bugfix:1]: This is a bit vague, I'm going to edit it to clarify that it's about the migration.

```suggestion
Make the `migrate_to_volto` process more robust when running multiple times or when plone.volto was installed first. [pbauer]
```

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

File: news/97.bugfix (hunk 1 above)
Comment (pbauer) [reply news/97.bugfix]: thanks

**Comment by pbauer** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**